notifications_utils/version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "44.1.0"
+__version__ = "45.0.0"
```

Comment:
Probably version `46.0.0` as `45.0.0` is taken at the moment for the custom tags.